### ISSUE TITLE
grad_bias should multiple learning_rate?

### ISSUE THREAD

**Issue by zhuyeqing** [hans/glove.py#1]
```
    # Compute gradients for bias terms
    grad_bias_main = weight * cost_inner
    grad_bias_context = weight * cost_inner
    # # in stanford c version ：should multiple learning_rate
    # grad_bias_main = weight * cost_inner * learning_rate
    # grad_bias_context = weight * cost_inner * learning_rate
```

**Comment by hans** [comment]
The learning rate is incorporated into the AdaGrad updates in [the next code section](https://github.com/hans/glove.py/blob/master/glove.py#L259):

``` python
        b_main -= (learning_rate * grad_bias_main / np.sqrt(gradsq_b_main))
        b_context -= (learning_rate * grad_bias_context / np.sqrt(
                gradsq_b_context))
```
